### PR TITLE
[ODS-4234] Add postman tests for EdOrg types added by an Extension 

### PR DESCRIPTION
**Pull request by semalaiappan**
I added postman tests for EdOrg types added by an Extension 

I also added disabled postman scripts for ODS-4637 which is used as manual run in future

### CODE REVIEW

File: Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,3212 @@
...
+							"script": {
+								"id": "2e06fb2c-bf20-407a-8af3-2b64943856cb",
+								"exec": [
+									"pm.test(\"Status code is 200\", () => {",
+									"        pm.expect(pm.response.code).to.equal(200);",
+									"    });",
+									"",
+									"pm.test(\"Should return some data for verification\", () => {",
+									"    var responseItems = pm.response.json();",
+									"     pm.expect(responseItems.endDate).to.be.equal(pm.environment.get(\"endDate\"));",
```
Comment (blmeyers) [inline Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json:2298]:
```suggestion
									"    var responseItem = pm.response.json();",
									"     pm.expect(responseItem.endDate).to.be.equal(pm.environment.get(\"endDate\"));",
```

File: Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,3212 @@
...
+							"script": {
+								"id": "44b732bc-5b3a-4ff3-b417-f9d27696793e",
+								"exec": [
+									"pm.test(\"Status code is 200\", () => {",
+									"        pm.expect(pm.response.code).to.equal(200);",
+									"    });",
+									"",
+									"pm.test(\"Should return some data for verification\", () => {",
+									"    var responseItems = pm.response.json();",
+									"     pm.expect(responseItems.staffReference.staffUniqueId).to.be.equal(pm.environment.get(\"known:staffUniqueId\"));",
```
Comment (blmeyers) [inline Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json:2508]:
```suggestion
									"    var responseItem = pm.response.json();",
									"     pm.expect(responseItem.staffReference.staffUniqueId).to.be.equal(pm.environment.get(\"known:staffUniqueId\"));",
```

File: Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,3212 @@
...
+							"script": {
+								"id": "1d355378-3326-46dc-a127-533ea04f1d86",
+								"exec": [
+									"pm.test(\"Status code is 200\", () => {",
+									"        pm.expect(pm.response.code).to.equal(200);",
+									"    });",
+									"",
+									"pm.test(\"Should return some data for verification\", () => {",
+									"    var responseItems = pm.response.json();",
+									"     pm.expect(responseItems.gradeLevels[0].gradeLevelDescriptor).to.be.equal(pm.environment.get(\"gradeLevelDescriptor\"));",
```
Comment (blmeyers) [inline Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json:2631]:
```suggestion
									"    var responseItem = pm.response.json();",
									"     pm.expect(responseItem.gradeLevels[0].gradeLevelDescriptor).to.be.equal(pm.environment.get(\"gradeLevelDescriptor\"));",
```

File: Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json [hunk 4]
```diff
@@ -0,0 +1,3212 @@
...
+							"script": {
+								"id": "77e4b68e-c3ec-413a-9070-ba11957ebf6b",
+								"exec": [
+									"pm.test(\"Status code is 200\", () => {",
+									"        pm.expect(pm.response.code).to.equal(200);",
+									"    });",
+									"",
+									"pm.test(\"Should return some data for verification\", () => {",
+									"    var responseItems = pm.response.json();",
+									"     pm.expect(responseItems.staffReference.staffUniqueId).to.be.equal(pm.environment.get(\"known:staffUniqueId\"));",
```
Comment (blmeyers) [inline Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json:2846]:
```suggestion
									"    var responseItem = pm.response.json();",
									"     pm.expect(responseItem.staffReference.staffUniqueId).to.be.equal(pm.environment.get(\"known:staffUniqueId\"));",
```

**Review by blmeyers — CHANGES_REQUESTED** [review]
Mostly minor changes, but need to decide if we're testing get by id or get by example for verifying after the update.

File: Postman Test Suite/Ed-Fi ODS API v3.x (Local).postman_environment.disabled.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
+{
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply Postman Test Suite/Ed-Fi ODS API v3.x (Local).postman_environment.disabled.json]: File removed

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite - Publishing Feature.postman_collection.disabled [hunk 6]
```diff
@@ -0,0 +1,1043 @@
+{
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply Postman Test Suite/Ed-Fi ODS-API Integration Test Suite - Publishing Feature.postman_collection.disabled]: File removed

File: Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json [hunk 7]
```diff
@@ -0,0 +1,3212 @@
...
+								"tpdm",
+								"universities",
+								"{{known:universityGUID}}"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get a University by nameOfInstitution",
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json:858]: @blmeyers - I removed query string parameter and looping response array to check nameOfInstitution

File: Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json [hunk 8]
```diff
@@ -0,0 +1,3212 @@
...
+							"listen": "test",
+							"script": {
+								"id": "003fef77-7b10-465b-ab8d-e1055a87ec7e",
+								"exec": [
+									"pm.test(\"Status code is 200\", () => {",
+									"        pm.expect(pm.response.code).to.equal(200);",
+									"    });",
+									"",
+									"pm.test(\"Should return some data for verification\", () => {",
+									"    var responseItems = pm.response.json();",
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json:880]: @blmeyers - I removed query string parameter and looping response array to check nameOfInstitution

File: Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json [hunk 9]
```diff
@@ -0,0 +1,3212 @@
...
+					}
+				}
+			],
+			"protocolProfileBehavior": {}
+		},
+		{
+			"name": "TeacherPreparationProviderProgram",
+			"item": [
+				{
+					"name": "Clean up  teacherCandidateTeacherPreparationProviderProgramAssociation Data",
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json:1329]: I removed eaning up TeacherCandidateTeacherPreparationProviderProgramAssociation data

File: Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json [hunk 10]
```diff
@@ -0,0 +1,3212 @@
...
+								"tpdm",
+								"teacherPreparationProviders",
+								"{{known:teacherPreparationProviderGUID}}"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get a teacherPreparationProvider by nameOfInstitution",
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json:1123]: @blmeyers   I removed query string parameter and looping response array to check nameOfInstitution

File: Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json (hunk 8 above)
[the remark this answers is not in the document]
Comment (blmeyers) [reply Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json:821]: You removed too much here.  Now you're just doing a get all, which might work for now but if we ever add more sample data you run the risk of your added entry not coming back on an initial request and requiring paging to reach.  What you had before was fine, you were just mixing two approaches.  This should likely be just using GetById (so pass the Id in the route) and then verify that the item got updated.  We're not trying to test GetAll or GetByExample for these test cases. Same for the other places you did get verification.

**Review by blmeyers — CHANGES_REQUESTED** [review]

File: Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json (hunk 8 above)
[the remark this answers is not in the document]
Comment (semalaiappan) [reply Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json]: @blmeyers   I updated to based on By ID and validate the name of institution in response result . Thanks

File: Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json (hunk 7 above)
[the remark this answers is not in the document]
Comment (semalaiappan) [reply Postman Test Suite/Ed-Fi ODS-API TPDM Test Suite.postman_collection.json]: @blmeyers   I updated to based on By ID and validate the name of institution in response result . Thanks

**Comment by semalaiappan** [comment]
@blmeyers - Please review it again . Thank you !

**Review by blmeyers — APPROVED** [review]
Looks good, just waiting on the last build to finish.

**Pull request merged**